sources/Platform/MacOS/MacOSWindow.mm

```diff
@@ -496,23 +496,25 @@ static NSUInteger GetNSWindowStyleMask(const WindowDescriptor& desc)
 
 void MacOSWindow::OnProcessEvents()
 {
-    NSEvent* event = nil;
+    @autoreleasepool {
```

Comment:
Same here as above. Please adjust the braces to the Allman coding style, sqaush your commits and I'll merge the PR.